ssl_league_protobufs/proto/ssl_vision_geometry.proto

```diff
@@ -4,71 +4,134 @@ option go_package = "github.com/RoboCup-SSL/ssl-game-controller/internal/app/vis
+    required float a2 = 5;
+    // Thickness of the arc.
+    required float thickness = 6;
+    // The type of this shape
+    optional SSL_FieldShapeType type = 7;
 }
 
 message SSL_GeometryFieldSize {
-  required int32 field_length = 1;
-  required int32 field_width = 2;
-  required int32 goal_width = 3;
-  required int32 goal_depth = 4;
-  required int32 boundary_width = 5;
-  repeated SSL_FieldLineSegment field_lines = 6;
-  repeated SSL_FieldCicularArc field_arcs = 7;
+    required int32 field_length = 1;
+    required int32 field_width = 2;
+    required int32 goal_width = 3;
+    required int32 goal_depth = 4;
+    required int32 boundary_width = 5;
+    repeated SSL_FieldLineSegment field_lines = 6;
+    repeated SSL_FieldCircularArc field_arcs = 7;
+    optional int32 penalty_area_depth = 8;
+    optional int32 penalty_area_width = 9;
+    optional int32 center_circle_radius = 10;
+    optional int32 line_thickness = 11;
+    optional int32 goal_center_to_penalty_mark = 12;
+    optional int32 goal_height = 13;
+    optional float ball_radius = 14;
+    optional float max_robot_radius = 15;
 }
 
 message SSL_GeometryCameraCalibration {
-  required uint32 camera_id     = 1;
-  required float focal_length = 2;
-  required float principal_point_x = 3;
-  required float principal_point_y = 4;
-  required float distortion = 5;
-  required float q0 = 6;
-  required float q1 = 7;
-  required float q2 = 8;
-  required float q3 = 9;
-  required float tx = 10;
-  required float ty = 11;
-  required float tz = 12;
-  optional float derived_camera_world_tx = 13;
-  optional float derived_camera_world_ty = 14;
-  optional float derived_camera_world_tz = 15;
+    required uint32 camera_id = 1;
+    required float focal_length = 2;
+    required float principal_point_x = 3;
+    required float principal_point_y = 4;
+    required float distortion = 5;
+    required float q0 = 6;
+    required float q1 = 7;
+    required float q2 = 8;
+    required float q3 = 9;
+    required float tx = 10;
+    required float ty = 11;
+    required float tz = 12;
+    optional float derived_camera_world_tx = 13;
+    optional float derived_camera_world_ty = 14;
+    optional float derived_camera_world_tz = 15;
+    optional uint32 pixel_image_width = 16;
+    optional uint32 pixel_image_height = 17;
+}
+
+// Two-Phase model for straight-kicked balls.
+// There are two phases with different accelerations during the ball kicks:
+// 1. Sliding
+// 2. Rolling
+// The full model is described in the TDP of ER-Force from 2016, which can be found here:
+// https://ssl.robocup.org/wp-content/uploads/2019/01/2016_ETDP_ER-Force.pdf
+message SSL_BallModelStraightTwoPhase {
+    // Ball sliding acceleration [m/s^2] (should be negative)
+    required double acc_slide = 1;
+    // Ball rolling acceleration [m/s^2] (should be negative)
+    required double acc_roll = 2;
+    // Fraction of the initial velocity where the ball starts to roll
+    required double k_switch = 3;
+}
```

Comment:
woah, do they just give us the friction values now?

Reply:
Apparently? It's not totally clear to me why these are part of the league protobufs. Is the vision server going to actually send meaningful values for these?